src/components/challengeWidget/challengeWidget.js

```diff
@@ -0,0 +1,39 @@
+import { Link } from "@reach/router";
+import React from "react";
+
+import "./challengeWidget.css";
+
+const ChallengeWidget = ({ title, label, linkTo = "#" }) => (
+  <div className="challengeWidget">
+    <Link to={linkTo}>
+      <h1>{title}</h1>
+      <Link to={label} />
+    </Link>
+    <div className="status">
+      <button>+</button>
+    </div>
+    <div className="Challenge_tags">
+      <button>Learning</button>
```

Comment:
I see that you created a `<Label>` component, you should use that here instead of these buttons.